crates/net/network/src/transactions/mod.rs

```diff
@@ -919,11 +919,18 @@ where
 
             for tx in transactions {
                 // recover transaction
-                let tx = if let Ok(tx) = tx.try_into_ecrecovered() {
-                    tx
-                } else {
-                    has_bad_transactions = true;
-                    continue
+                let tx = match tx.try_into_ecrecovered() {
+                    Ok(tx) => tx,
+                    Err(badtx) => {
+                        trace!(target: "net::tx",
+                            %peer_id,
```

Comment:
```suggestion
                            peer_id=format!("{peer_id:#}"),
```

shortens it, takes up less log space. we keep the hash long because we can search for it in block explorer then.